project/Build.scala

```diff
@@ -167,10 +167,12 @@ object ScalaMeterBuild extends MechaRepoBuild {
       "org.apache.commons" % "commons-math3" % "3.2",
       "org.scala-lang" % "scala-reflect" % "2.11.0",
       "org.scala-lang.modules" %% "scala-xml" % "1.0.1",
-      "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.1"
+      "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.1",
```

Comment:
Since persistors are not defined in the core package, we should move this to the dependencies of the main project.